packages/nextjs/components/CheckinForm.tsx

```diff
@@ -76,66 +77,54 @@ const CheckinForm = ({ latLng = [0, 0] }) => {
-                  <input
-                    type="number"
-                    name="coordinateInputY"
-                    // placeholder={formValues.coordinateInputY.toString()}
-                    className="input w-full max-w-xs"
-                    value={latLng[1]}
-                    onChange={handleChange}
-                  />
-                  )
-                </label>
-                <label className="flex center flex-row items-center">
-                  <div>Timestamp:</div>
-                  {/* Not Sure we input this, might be on EAS.sol in the case of onchain attestations */}
-                  <input
-                    type="number"
-                    name="timestamp"
-                    value={formValues.timestamp}
-                    onChange={handleChange}
-                    className="input w-full max-w-xs"
-                  />
-                </label>
-                <label className="flex center flex-row items-center w-full">
-                  <div>Data:</div>
-                  {/* Not Sure we input this, might be on EAS.sol in the case of onchain attestations */}
-                  <input
-                    type="text"
-                    name="data"
-                    value={formValues.data}
-                    placeholder={formValues.data}
-                    className="input w-full"
-                    onChange={handleChange}
-                  />
-                </label>
+          <form onSubmit={handleSubmit} className="card m-5 flex-col gap-2 ">
+            <label className="flex center flex-row items-center flex-grow gap-3">
+              <Image src="/location.svg" alt="location_svg" width="50" height="50" />
+              <input
+                type="number"
+                name="coordinateInputX"
+                // placeholder={formValues.coordinateInputX.toString()}
+                className="input input-bordered w-full  bg-base-200 border-indigo-500 text-black"
+                value={latLng[0]}
+                onChange={handleChange}
+              />
+              <input
+                type="number"
+                name="coordinateInputY"
+                // placeholder={formValues.coordinateInputY.toString()}
+                className="input input-bordered w-full  bg-base-200  border-indigo-500 text-black"
+                value={latLng[1]}
+                onChange={handleChange}
+              />
+            </label>
+            <label className="flex center flex-row flex-grow items-center gap-2">
+              {/* Not Sure we input this, might be on EAS.sol in the case of onchain attestations */}
+              <Image src="/datetime.svg" alt="datetime" width="30" height="30" />
 
-                <input type="submit" value="Check-in" className="input btn btn-primary" />
-              </div>
-            </div>
+              <input
+                type="number"
+                name="timestamp"
+                value={formValues.timestamp}
```

Comment:
This should be converted to a human-readable date + time in the UI, but yes the Unix timestamp should be recorded on chain.

I also think we may need a `timezone` attribute in the schema ...